src/components/Book.css

```diff
@@ -1,11 +1,13 @@
 .bookSection {
-  width: 80%;
+  width: 79%;
   height: 9rem;
-  margin: 2.313rem 6.25rem 0;
-  padding: -4rem 6.188rem 1.625rem 1.688rem;
-  border-radius: 4px;
+  padding: -4rem 6.17rem 1.625rem 6.25rem;
+  border-radius: 6px;
   border: solid 1px #e8e8e8;
   background-color: #fff;
+  margin-left: auto;
+  margin-right: auto;
+  margin-top: 1rem;
 }
 
 .bookContainer {
```

Comment:
- [ ] As requested by the previous reviewer, Please make sure your styling is EXACTLY the same as the one provided on [Zeplin](https://zeplin.io/). Actually your your header, main and form sections do not have the the same widths let's correct this 👍🏻  

![Screenshot](https://user-images.githubusercontent.com/79058364/201452371-c9b60016-d350-4367-8b3d-0162c5f8cd79.png)